specification/containerservices/resource-manager/Microsoft.ContainerService/preview/2018-09-30-preview/openShiftManagedClusters.json

```diff
@@ -466,10 +536,7 @@
         "count": {
           "type": "integer",
           "format": "int32",
-          "maximum": 5,
-          "minimum": 1,
-          "description": "Number of agents (VMs) to host docker containers. Allowed values must be in the range of 1 to 5 (inclusive). The default value is 2. ",
-          "default": 2
+          "description": "Number of agents (VMs) to host docker containers. Allowed values must be in the range of 1 to 5 (inclusive). The default value is 2. "
```

Comment:
If the allowed values are still 1-5, why remove the minimum and maximum?

Reply:
Good Catch ! I'll update the description since we removed the limitations.